### DISCUSSION TITLE
How to fix BarGraphItem height when zoom in or zoom out plot

### DISCUSSION THREAD

**Discussion by mike9755** [pyqtgraph/pyqtgraph#2256] [General]
Hi dears:
Is it possible to fix the height of BarGraphItem to fixed percent of window height? Means shows fixed heith to the window when zoom in or zoom out the BarGraphItem. It should be like text which height is pixels and never changed in zoom mode, hope someone can support, thank you very much.

```python
import numpy as np

import pyqtgraph as pg

pg.setConfigOption('leftButtonPan', False) 

win = pg.plot()
win.setWindowTitle('pyqtgraph example: BarGraphItem')

x = np.arange(10)

bg1 = pg.BarGraphItem(x=x, height=5, width=1,brush='r')

win.addItem(bg1)
win.setYRange(0,110)

win.setLimits(yMin=0,yMax=110,xMin=-0.5,xMax=11)

if __name__ == '__main__':
    pg.exec() 
```

![image](https://user-images.githubusercontent.com/5673534/160400063-f3de8c04-9579-4513-b983-487786d0ead6.png)

**Comment by ixjlyons** [comment]
Would disabling mouse zoom on the y axis altogether achieve what you're looking to do? You can do that with `win.setMouseEnabled(y=False)`.

If you do still want to be able to change the y axis range, you'd have to connect to `sigYChanged` from the `ViewBox` and work out the right height to set. It seems like there might be a simpler way to do what you're ultimately trying to achieve though.
